muffnn/mlp/tests/test_mlp_classifier.py

```diff
@@ -95,9 +95,8 @@ def check_multilabel_predictions_na(clf, X, y):
 def check_error_with_nans(clf, X, y):
     with pytest.raises(ValueError) as excinfo:
         clf.fit(X, y).predict(X)
-    assert "Input contains NaN, infinity or a value too large for \
-        dtype('float64')." in str(excinfo.value)
-
+    assert "Input contains NaN, infinity or a value too large for " \
```

Comment:
You don't need the backslash here. Wasn't this test failure related to the fact that muffnn raises an AssertionError and not a ValueError? I might have misread circle.